src/main/java/me/ryanhamshire/GriefPrevention/BlockEventHandler.java

```diff
@@ -285,6 +293,36 @@ public void onBlockPlace(BlockPlaceEvent placeEvent)
         //if the block is being placed within or under an existing claim
         PlayerData playerData = this.dataStore.getPlayerData(player.getUniqueId());
         Claim claim = this.dataStore.getClaimAt(block.getLocation(), true, playerData.lastClaim);
+
+        // Check for double chests placed just outside the claim boundary
+        if (block.getBlockData() instanceof Chest)
+        {
+            for (BlockFace face : HORIZONTAL_DIRECTIONS)
+            {
+                Block relative = block.getRelative(face);
+                if (!(relative.getBlockData() instanceof Chest)) continue;
+
+                Claim relativeClaim = this.dataStore.getClaimAt(relative.getLocation(), true, playerData.lastClaim);
```

Comment:
Should use `claim` here instead of `playerData.lastClaim`, odds are much higher that chest is placed in same claim as an adjacent block.

Reply:
Added!